lib/approve.js

```diff
@@ -1,24 +1,19 @@
 'use strict';
 var github = require('./github');
 
-module.exports = function(number, reason) {
-    return github.get('/repos/:owner/:repo/pulls/:number/reviews', { number: number })
-        .then(function(reviews) {
-            var isReviewed = reviews.some(function(r) {
-                return r.user.login == 'wpt-pr-bot';
-            });
+module.exports = async function(number, reason) {
+    const reviews = await github.get('/repos/:owner/:repo/pulls/:number/reviews', { number: number });
 
-            if (isReviewed) {
-                return false;
-            }
+    var isReviewed = reviews.some(r => { return r.user.login == 'wpt-pr-bot'; });
```

Comment:
```suggestion
    const isReviewed = reviews.some(r => { return r.user.login == 'wpt-pr-bot'; });
```

Might as well make this more modern?